build.py

```diff
@@ -126,9 +131,8 @@ def validate_package():
     # cache overshadowing actual changes made.
     current_platform = utility.get_current_platform()
 
-    mssqlcli_wheel_name = [
-        pkge for pkge in mssqlcli_wheel_dir if current_platform in pkge and
-        'dev-latest' not in pkge]
+    mssqlcli_wheel_name = [pkge for pkge in mssqlcli_wheel_dir
+                           if current_platform in pkge and 'dev-latest' not in pkge]
```

Comment:
I want to remove this dev-latest concept - it was a bad idea. But that's for another PR